SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -69,6 +70,8 @@ void USpatialLatencyTracer::RegisterProject(UObject* WorldContextObject, const F
 	std::cerr.rdbuf(&Stream);
 
 	StdoutExporter::Register();
+
+	GetTracer(WorldContextObject)->MessagePrefix = MessagePrefix;
```

Comment:
```suggestion
	MessagePrefix = MessagePrefix;
```

Reply:
`USpatialLatencyTracer::RegisterProject` is a static method, so it doesn't have a local MessagePrefix variable.

Reply:
Oh I see my bad, might want to check the return then as GetTracer can return NULL